InnerEye/Common/fixed_paths.py

```diff
@@ -25,6 +25,20 @@ def repository_root_directory(path: Optional[PathOrString] = None) -> Path:
         return root
 
 
+def repository_parent_directory(path: Optional[PathOrString] = None) -> Path:
+    """
+    Gets the full path to the parent directory that holds the present repository.
+    :param path: if provided, a relative path to append to the absolute path to the repository root.
+    :return: The full path to the repository's root directory, with symlinks resolved if any.
+    """
+    current = Path(__file__)
+    root = current.parent.parent.parent.parent
```

Comment:
[Suggestion] You could do `root = repository_root_directory().parent`.

Reply:
+1 on that. paths are awkward to main, so re-using is safer

Reply:
done